_posts/2017-06-11-RealityWinner.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: "Solidarity with #RealityWinner; accused leaker of NSA material"
```

Comment:
Skip the semi-colon in the title, e.g.

`
title: "Solidarity with #RealityWinner; accused leaker of NSA material" -> title: "Solidarity with accused leaker of NSA material #RealityWinner"
`